### DISCUSSION TITLE
Need example of date-picker table without date input

### DISCUSSION THREAD

**Discussion by wilcoschoneveld** [chakra-ui/ark#1745] [Q&A]
Hello!

I would like an example of the date picker component with `"@ark-ui/react": "1.0.1"`

I've tried to construct it myself but the datepicker components seem to rely on a trigger and only then show

Requirements:
- Should show date picker table immediately (not in a popup)
- No input box needed, just the grid/table
- No need for positioner or portal, just render

If possible I would also like an example of two months side by side

Could you post a code snippet please? Or point me to the right direction.

Thanks!

**Comment by cschroeter** [comment]
@wilcoschoneveld 

Something like this?

https://github.com/chakra-ui/ark/assets/1846056/42387a11-62a0-4031-a0fb-aad6c635c08a

https://github.com/chakra-ui/ark/commit/9b52199f40a462de793497fba7769f64e4ee6fc8#diff-8f660d2bf0c3095b76165aa2bd41dcd482cc6f5a31294f976ba9cca5e0ae9f23

**Reply by wilcoschoneveld** [reply]
@cschroeter awesome, thanks a lot!

**Reply by wilcoschoneveld** [reply]
@cschroeter 

when using:
```
    <DatePicker.Root open={true}>
    </DatePicker.Root>
```

I get a warning in the console:
```
[@zag-js/dismissable] node is `null` or `undefined` 
```

is that a problem?

**Reply by cschroeter** [reply]
No. The `DatePicker` machine is looking for a `data-part="content"` which is not available. You could add the `Content` part and f orce the `present` on the `DatePicker.Root` but unfortunately we have a bug there :D So please just ignore the warning. I will let you know when there is a fix


```tsx
<DatePicker.Root
      open={true}
      numOfMonths={2}
      selectionMode="range"
      closeOnSelect={false}
      present
    >
      {(api) => {
        const offset = api.getOffset({ months: 1 })
        return (
          <DatePicker.Content>
          /* ... */ 
          
  ```

**Reply by wilcoschoneveld** [reply]
Alright! Thanks for the quick answer

**Reply by wilcoschoneveld** [reply]
@cschroeter Unfortunately the proposed solution is not meeting the requirements for us because it breaks the accessibility features of the date picker. I've tested a bit and it seems I need the `DatePicker.Content` for keyboard navigation to work properly. But this introduces the bug that clicking outside the datepicker hides it. Any suggestions?

**Reply by cschroeter** [reply]
This will be fixed when https://github.com/chakra-ui/zag/issues/1027 is resolved. Let me see if I can come up with a workaround.
